releases/1.0.0b.md

```diff
@@ -0,0 +1,10 @@
+## Arches Controlled Lists 1.0.0beta Release Notes
```

Comment:
Is there a PR planned to spruce up the project metadata? (e.g. change `"Development Status :: 3 - Alpha",` and add authors and urls, or is that for Monday?)

See archesproject/arches-modular-reports@9cefacf517951e9d831195e482ed7f6e3efb7cf5 minus the bits about 7.6 compat

Reply:
thanks - I took a stab at updating the metadata here